linapple.conf

```diff
@@ -1,253 +0,0 @@
-#
-# linapple.conf - config file used by LinApple, Apple][ (Apple2, Apple 2) emulator for Linux and other POSIX systems with SDL support
```

Comment:
I was wondering why this diff didn't show this file as being moved to `res/linapple.conf`, and it seems that it is not in fact deleted; it is still there in the repo, with size 0.  I think it should probably be OK to just delete it entirely?